astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -31,6 +38,33 @@
 from astronomer.providers.utils.typing_compat import Context
 
 
+def _check_queries_finish(conn: SnowflakeConnection, query_ids: list[str]) -> bool:
+    """Check whether snowflake queries finish (in aborting, failed_with_error or success)"""
+    with closing(conn) as conn:
+        for query_id in query_ids:
+            status = conn.get_query_status(query_id)
+
+            if status == QueryStatus.ABORTING:
+                raise AirflowException(f"ABORTING {ABORTING_MESSAGE}")
```

Comment:
It might help the user if we also log the query ID in the exception.

Reply:
Agree. I was meant to match the original message. I just updated both places. Thanks!